docs/utilization.md

```diff
@@ -2,157 +2,201 @@
+
+```
+TARGET_BILLABLE_HOURS = (HOURS_IN_WEEK - HOURS_NOT_WORKED) * BILLING_EXPECTATION
+```
+
+### Billing expectation
+The percent of [hours in a week](#hours-in-a-week) which are expected to be billable for a given reporting period. Set per user, per timecard, and will vary as an individual's role, position, or organization change. See [the handbook](https://handbook.tts.gsa.gov/tock/#weekly-billable-hour-expectations) for additional details on billing expectations.
 
 **Default: 80%**
 
+### Hours in a week
+The number of hours in a standard week, i.e., `40`. This value is static.
+
+### Non-billable employee
+An employee is considered non-billable if their [target billable hours](#target-billable-hours) over a time period is 0. Utilization is not calculated for these individuals because we're currently unable to <a href="https://en.wikipedia.org/wiki/Division_by_zero">divide by zero</a>.
+
+All billable hours and allocation tocked by non-billable employees are included in aggregations for their respective business unit and organization.
 ## Example
 
-Stacey is an active and billable 18F employee, here's a look at their recent tock activity! As an individual contributor, Stacey has a `BILLING_EXPECTATION` of 80%.
+Stacey is an active and billable 18F employee, here's a look at their recent tock activity! As an individual contributor, Stacey has a billing expectation of 80%.
 
 Project | Week 1 | Week 2 | Week 3 | Week 4
 --------|--------|--------|--------|--------
 Out of Office| 0.0 | 8.0 | 0.0 | 0.0
-Billable | 24.0 | 28 | 32.0 | 32.0
-Non-Billable | 16.0 | 4.0 | 8.0 | 8.0
+Hourly Billable | 24.0 | 30.0 | 0.0 | 32.0
+Weekly Billable | 25% | 0% | 100% | 0%
+Hourly Non-Billable | 8.0 | 2.0 | 8.0 | 8.0
 
 ### Week 1
-Stacey's `utilization` for Week 1 is 75%.
-Stacey tocked 24 of an expected 32 billable hours.
+Stacey's `utilization` for Week 1 is 100%.
+
+Stacey tocked 24 billable hours to an hourly project and 8 billable hours to a weekly project, for a combined 32 hours of an expected 32 target billable hours.
 
 ```python
 # Week 1
 billable_hours = 24.0
-non_billable_hours = 16.0
+non_billable_hours = 8.0
+billable_allocation = .25
+out_of_office = 0
 billing_expectation = 0.80
 
-total_hours = billable_hours + non_billable_hours
-# 24.0 + 16.0 = 40
+target_billable_hours = (hours_in_week - out_of_office) * billing_expectation
+# (40 - 0) * 0.80 = 32
 
-target_billable_hours = total_hours * billing_expectation
-# 40 * 0.80 = 32
+billable_allocation_hours = billable_allocation * target_billable_hours
+# .25 * 32 = 8
 
-utilization = billable_hours / target_billable_hours
-# 24 / 32 = 0.75
+utilization = (billable_hours + billable_allocation_hours) / target_billable_hours
+# (24 + 8) / 32 = 1
 ```
 
 ### Week 2
```

Comment:
Suggestion: Add the table with the data before  each example, as it difficult to verify the math scrolling up and down constantly to check the numbers.

Reply:
@edwintorres Yeah, I see what you're saying. Before I make that change, do you have any concerns about this creating a need for the table to be updated in 4 places if the examples change in the future?

Reply:
Yes I see that as a problem, but how much do you think this example table is going to change?